include/meshio/obj.inl

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2017, Lakshman Anumolu, Pradeep Garigipati
+ * All rights reserved.
+ *
+ * This file is part of MeshIO whose distribution is governed by
+ * the BSD 2-Clause License contained in the accompanying LICENSE.txt
+ * file.
+ */
+
+/**
+ * Write mtl file, courtesy obj_writer.cc of tinyobjloader.
+ * TODO: Thorough testing needed for this function
+ */
+template<typename T>
+bool WriteMtl(const char* pMtlFileName,
```

Comment:
Lets go with `writeMaterial`. I don't remember exactly, didn't we agree for camel case for function naming?